charter.md

```diff
@@ -22,14 +22,20 @@ Server-initiated services are out of scope. The working group will first deliver
 a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
 IP proxying.
 
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP is important to ensure their continued useability.
+The working group will consider extensions to these documents that provide missing functionality or enhance performance
+for real world use cases. The initial set of extensions will be in support of UDP listening, QUIC-aware proxying, and
```

Comment:
We might want to define what a "real world use case" is more formally now to avoid arguing about it later. It looks like it's used twice in this new charter and is the gating factor for new work so it's bound to come up

Reply:
I think that's a good idea, but I don't really have a great definition for "real world use cases." I'll think on this.

Reply:
Since use cases that are real world are ultimately driven by people who care about them and want to support them, perhaps we can say something like "use cases with implementation and deployment proponents"?

Reply:
(I modified the suggestion to use this phrasing)

Reply:
The extension mechanisms are decently protected from observer meddling. Folks can use and experiment with the extension points without strong coordination from a standards body.

I'm guessing we're mainly interested in adopting work to the WG (on a standards track?) that has sufficient interest from the WG beyond just single proponent or deployment.

Reply:
Right -- hence plural "proponents". Happy to take alternative phrasings if they're more clear.

Reply:
Ah, I think my reading of the multiple plurals in one sentence did not grasp that intent then. How about something like

> As such, the working group will consider adopting an extension to these documents when it provides missing
functionality, or otherwise eases deployability for use cases, where there are multiple implementation and/or deployment proponents.

Reply:
That reads good to me. @ekinnear, if you agree, could you please merge into the suggestion above?

Reply:
To be super pedantic, are we saying: 

- when it provides missing functionality, or otherwise eases deployability for
use cases, where there are multiple implementation and/or deployment proponents.

- when it provides missing functionality, otherwise eases deployability for
use cases, or where there are multiple implementation and/or deployment proponents.

- when it provides missing functionality or otherwise eases deployability for
use cases, where there are multiple implementation and/or deployment proponents.

I think the third one is correct comma usage + says (functionality OR deployability) AND (implementation proponents OR deployment proponents)

The second is all ORs everywhere, which was not my reading of the intent, but wouldn't be horrible either.

(updating to have the third bullet in the suggestion, but happy to adjust as necessary

Reply:
Used bullet three in [8af81ca](https://github.com/ietf-wg-masque/wg-materials/pull/7/commits/8af81ca9873dc5f769a87707af732f24be6455bf)